src/Microsoft.OData.Core/UriParser/Parsers/ODataPathParser.cs

```diff
@@ -86,6 +87,20 @@ internal static void ExtractSegmentIdentifierAndParenthesisExpression(string seg
                 identifier = segmentText;
                 parenthesisExpression = null;
             }
+            else if (segmentText[segmentText.Length - 1] == ':')
+            {
+

```

Comment:
Remove Blank Space

Reply:
@Sreejithpin We usually follow closing parenthesis with a blank line if it is the last else clause. Look at some of the other existing code